libs/packages/layouts/src/lib/feature/header/header.component.scss

```diff
@@ -0,0 +1,11 @@
+.sds-navbar-blank {
+  min-height: 3rem;
+}
+
+.sds-nav__secondary-blank {
+  bottom: 3rem;
+}
+
+.usa-navbar {
+  justify-content: flex-end;
+}
```

Comment:
https://github.com/GSA/sam-styles/pull/368/files

Reply:
@Bibek-Regmi  are you still waiting on sam-styles release for it

Reply:
Yes - I got some feedback on the sam-styles PR, and updated it in sam-styles.

Reply:
@Bibek-Regmi  Once sam-styles done, updated the sam-styles version and verify changes